src/applications/simple-forms/21-0972/pages/preparerQualifications1.js

```diff
@@ -0,0 +1,33 @@
+import { preparerQualificationsQuestionTitle } from '../config/helpers';
+import GroupCheckboxWidget from '../../shared/components/GroupCheckboxWidget';
+import { preparerQualificationsOptions } from '../definitions/constants';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    preparerQualifications: {
+      'ui:widget': GroupCheckboxWidget,
+      'ui:options': {
+        updateSchema: formData => {
+          // TODO: figure out why this isn't populating correctly
+          return {
+            title: preparerQualificationsQuestionTitle(formData),
+          };
+        },
+        forceDivWrapper: true,
+        // TODO: build out detailed info for each option
+        labels: Object.values(preparerQualificationsOptions),
```

Comment:
the sketch has more info for each checkbox. Any suggestions?
<img width="338" alt="Screenshot 2023-06-26 at 12 03 22 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/57802560/c5dd624e-1987-4dd4-965b-574f4f954184">

Reply:
I don't even see this design supported here: https://design.va.gov/storybook/?path=%2Fdocs%2Fcomponents-va-checkbox-group--default. So I don't know where the designer pulled that from. Maybe they got confused with Radio? https://design.va.gov/storybook/?path=%2Fdocs%2Fcomponents-va-radio--default#tile

Reply:
Discussing options with design